Dockerfile.prometheus

```diff
@@ -1,31 +1,27 @@
 FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_golang_1.22 as builder
-
 RUN dnf module install -y nodejs:16/development
 ENV NPM_CONFIG_NODEDIR=/usr
+WORKDIR /opt/app-root/src
 
-RUN yum install -y --setopt=tsflags=nodocs bzip2 jq golang-github-prometheus-promu
+RUN yum install -y --setopt=tsflags=nodocs bzip2 jq
 
 COPY prometheus/ .
 
-WORKDIR /workspace
-
 ENV GOFLAGS='-mod=mod'
-ARG GO_BIN=ARG GO_BIN=/workspace/prometheus/deps/gomod/bin
+RUN go install -mod=mod github.com/prometheus/promu@v0.15.0
+RUN make build
 
-# Move the prometheus-promu we install to go's bin/ to avoid re-installing it during the build
-# Makefile.common expects the promu binary in a certain location
-RUN mkdir $GO_BIN \
-    && ln -s $(which promu) $GO_BIN/promu \
-    && make build
+RUN ls -l /opt/app-root/src/
```

Comment:
Is this just left over debug or what's the purpose?

Reply:
oh, forgot to remove that, done now ;)